<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x0:0, y0:0, x2:2, y2:2, stop:0 rgba(35, 31, 30, 255), stop:1 rgba(23, 26, 42, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>360</width>
      <height>720</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>480</y>
        <width>471</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">

	border-radius: 7px;
	font: 700 36pt &quot;Constantia&quot;;
	font-style: normal;
	color: #FFFFFF;</string>
      </property>
      <property name="text">
       <string>Нет подключения..</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>341</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	box-sizing: border-box;
	border: 3px solid #202252;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(32, 38, 61, 255), stop:1 rgba(49, 35, 33, 255));
	border-radius: 7px;
	font: 700 36pt &quot;Constantia&quot;;
	font-style: normal;
	color: #FFFFFF;
}

QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 255));
}</string>
      </property>
      <property name="text">
       <string>Играть</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>291</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="res.qrc">:/res/logo.png</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>341</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	box-sizing: border-box;
	border: 3px solid #202252;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(32, 38, 61, 255), stop:1 rgba(49, 35, 33, 255));
	border-radius: 7px;
	font: 700 36pt &quot;Constantia&quot;;
	font-style: normal;
	color: #FFFFFF;
}

QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 255));
}</string>
      </property>
      <property name="text">
       <string>Настройки</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>460</y>
        <width>341</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	box-sizing: border-box;
	border: 3px solid #202252;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(32, 38, 61, 255), stop:1 rgba(49, 35, 33, 255));
	border-radius: 7px;
	font: 700 36pt &quot;Constantia&quot;;
	font-style: normal;
	color: #FFFFFF;
}

QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 255));
}</string>
      </property>
      <property name="text">
       <string>Выход</string>
      </property>
     </widget>
     <zorder>label_3</zorder>
     <zorder>pushButton_2</zorder>
     <zorder>pushButton_10</zorder>
     <zorder>pushButton_11</zorder>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>308</width>
        <height>301</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="splitters">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>189</width>
           <height>10</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
	border-radius: 7px;
	font: 700 24pt &quot;Constantia&quot;;
	font-style: normal;
	color: #FFFFFF;

</string>
         </property>
         <property name="text">
          <string>ip Addres</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="styleSheet">
          <string notr="true">	border-radius: 7px;
	font-size: 20pt;
	font-style: normal;
	color: #FFFFFF;</string>
         </property>
         <property name="text">
          <string>localhost</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>189</width>
           <height>10</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
	border-radius: 7px;
	font: 700 24pt &quot;Constantia&quot;;
	font-style: normal;
	color: #FFFFFF;

</string>
         </property>
         <property name="text">
          <string>port</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="styleSheet">
          <string notr="true">	border-radius: 7px;
	font-size: 20pt;
	font-style: normal;
	color: #FFFFFF;</string>
         </property>
         <property name="text">
          <string>2323</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="styleSheet">
          <string notr="true">QPushButton {
	box-sizing: border-box;
	border: 3px solid #202252;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(32, 38, 61, 255), stop:1 rgba(49, 35, 33, 255));
	border-radius: 7px;
	font: 700 36pt &quot;Constantia&quot;;
	font-style: normal;
	color: #FFFFFF;
}

QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 255));
}</string>
         </property>
         <property name="text">
          <string>Создать игру</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>331</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	border-radius: 7px;
	font: 700 22pt &quot;Constantia&quot;;
	font-style: normal;
	color: #FFFFFF;</string>
      </property>
      <property name="text">
       <string>Создание игры</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>341</width>
        <height>281</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="splitters_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>189</width>
           <height>10</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
	border-radius: 7px;
	font: 700 24pt &quot;Constantia&quot;;
	font-style: normal;
	color: #FFFFFF;

</string>
         </property>
         <property name="text">
          <string>ip Addres</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="styleSheet">
          <string notr="true">	border-radius: 7px;
	font-size: 20pt;
	font-style: normal;
	color: #FFFFFF;</string>
         </property>
         <property name="text">
          <string>192.168.1.103</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>189</width>
           <height>10</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
	border-radius: 7px;
	font: 700 24pt &quot;Constantia&quot;;
	font-style: normal;
	color: #FFFFFF;

</string>
         </property>
         <property name="text">
          <string>port</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="styleSheet">
          <string notr="true">	border-radius: 7px;
	font-size: 20pt;
	font-style: normal;
	color: #FFFFFF;</string>
         </property>
         <property name="text">
          <string>2323</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="styleSheet">
          <string notr="true">QPushButton {
	box-sizing: border-box;
	border: 3px solid #202252;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(32, 38, 61, 255), stop:1 rgba(49, 35, 33, 255));
	border-radius: 7px;
	font: 700 28pt &quot;Constantia&quot;;
	font-style: normal;
	color: #FFFFFF;
}

QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 255));
}</string>
         </property>
         <property name="text">
          <string>Присоединиться</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>331</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	border-radius: 7px;
	font: 700 22pt &quot;Constantia&quot;;
	font-style: normal;
	color: #FFFFFF;</string>
      </property>
      <property name="text">
       <string>Подключение к игре</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>301</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	box-sizing: border-box;
	border: 3px solid #202252;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(32, 38, 61, 255), stop:1 rgba(49, 35, 33, 255));
	border-radius: 7px;
	font: 700 36pt &quot;Constantia&quot;;
	font-style: normal;
	color: #FFFFFF;
}

QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 255));
}</string>
      </property>
      <property name="text">
       <string>Создать</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>340</y>
        <width>301</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	box-sizing: border-box;
	border: 3px solid #202252;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(32, 38, 61, 255), stop:1 rgba(49, 35, 33, 255));
	border-radius: 7px;
	font: 700 28pt &quot;Constantia&quot;;
	font-style: normal;
	color: #FFFFFF;
}

QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 255));
}</string>
      </property>
      <property name="text">
       <string>Подключиться</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_7">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>361</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">

	border-radius: 7px;
	font: 700 25pt &quot;Constantia&quot;;
	font-style: normal;
	color: #FFFFFF;</string>
      </property>
      <property name="text">
       <string>Нет подключения..</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>341</width>
        <height>341</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>490</y>
        <width>131</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	border-radius: 7px;
	font: 700 24pt &quot;Constantia&quot;;
	font-style: normal;
	color: #004220;
	
</string>
      </property>
      <property name="text">
       <string>Сходить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_14">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>490</y>
        <width>131</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
	border-radius: 7px;
	font: 700 24pt &quot;Constantia&quot;;
	font-style: normal;
	color: #FFFFFF;

</string>
      </property>
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_15">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>650</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	border-radius: 7px;
	font: 700 24pt &quot;Constantia&quot;;
	font-style: normal;
	color: #991400;
	
</string>
      </property>
      <property name="text">
       <string>Выход</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>301</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	box-sizing: border-box;
	border: 3px solid #202252;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(32, 38, 61, 255), stop:1 rgba(49, 35, 33, 255));
	border-radius: 7px;
	font: 700 36pt &quot;Constantia&quot;;
	font-style: normal;
	color: #FFFFFF;
}

QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 255));
}</string>
      </property>
      <property name="text">
       <string>Создать</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>310</y>
        <width>301</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	box-sizing: border-box;
	border: 3px solid #202252;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(32, 38, 61, 255), stop:1 rgba(49, 35, 33, 255));
	border-radius: 7px;
	font: 700 28pt &quot;Constantia&quot;;
	font-style: normal;
	color: #FFFFFF;
}

QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 255));
}</string>
      </property>
      <property name="text">
       <string>Подключиться</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>840</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>840</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
